Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md

```diff
@@ -0,0 +1,94 @@
+# Upgrading python interpreter version from 3.7.10 to 3.8.12
+
+Oppia has updated it's python interpreter version 3.7.10 in a phase of switching to a more updated interpreter version 3.8.12. For effective and effortless switching to the newer version, this guide will provide a brief and detailed instruction on how to achieve just that. Please follow the below mentioned instructions carefully.
```

Comment:
it's --> its

Actually this seems to introduce a bunch of grammar mistakes. I thought I provided a clean version in my previous comment? (Please read it carefully.)

Reply:
Done !!